test/Datadog.Trace.Tests/ApiTests.cs

```diff
@@ -70,7 +70,7 @@ public async Task SendTracesAsync_500_ErrorIsCaught()
             sw.Stop();
 
             requestMock.Verify(x => x.PostAsync(It.IsAny<Span[][]>(), It.IsAny<FormatterResolverWrapper>()), Times.Exactly(5));
-            Assert.InRange(sw.ElapsedMilliseconds, 1000, 16000); // should be ~ 3200ms
+            Assert.InRange(sw.ElapsedMilliseconds, 1000, 20_000); // should be ~ 3200ms
```

Comment:
Should we even be checking the elapsed time in this test? I think it should only assert that a 500 http response doesn't cause an exception to be thrown.

Reply:
I was thinking the same thing originally. I've fixed this in 4af9b8bedb045d80f324ffff63eb1f8b992696d2